src/common/WorkQueue.cc

```diff
@@ -122,6 +122,8 @@ void ThreadPool::worker(WorkThread *wt)
 	  processing++;
 	  ldout(cct,12) << "worker wq " << wq->name << " start processing " << item
 			<< " (" << processing << " active)" << dendl;
+          assert(strlen(wq->thread_name) < 16);
+          pthread_setname_np(wt->get_thread_id(), wq->thread_name);
```

Comment:
Name is set when wq start its job, it could be set to "thread_pool" after end, but I left it like this. Changing back to thread_pool could mask  short wq jobs.